app/src/main/sentences/sl/lyrics.dslf

```diff
@@ -0,0 +1,5 @@
+lyrics: high
+pokaži|pridobi|naloži|kako se začne|poišči? mi? besedilo za|od|pesmi? .song.;
+zapoj pesem? .song.;
+.pesem. besedilo;
+zapoj|kako se začne|(kako se začne) (|tista ki se začne|govori o|poje o)|(z besedilom) .song.;
```

Comment:
There is a syntax error (the `|` after the `(`). Also, I am not sure what this sentence would mean: 
- the **piece** `zapoj|kako se začne|(kako se začne)` would match all of the following **pieces** of texts: "zapoj se zacne", "kako se zacne", "zapoj se kako se zacne", "kako se kako se zacne"
- `tista ki se začne|govori o|poje o` would match: "tista ki se začne o o", "tista ki se začne poje o", "tista ki se govori o o", "tista ki se govori poje o"